src/applications/disability-benefits/all-claims/utils.jsx

```diff
@@ -501,3 +504,28 @@ export const showPtsdAssaultConclusion = form =>
   form['view:upload781aChoice'] === 'answerQuestions' &&
   (_.get('view:selectablePtsdTypes.view:mstPtsdType', form, false) ||
     _.get('view:selectablePtsdTypes.view:assaultPtsdType', form, false));
+
+export const ancillaryFormUploadUi = (label, itemDescription) =>
+  fileUploadUI(label, {
+    itemDescription,
+    hideLabelText: !label,
+    fileUploadUrl: `${environment.API_URL}/v0/upload_supporting_evidence`,
+    fileTypes: ['pdf', 'jpg', 'jpeg', 'png'],
```

Comment:
Are we sure about these file types? The old list was longer and IIRC we got it from evss directly, but that was a while ago.

Reply:
It looks like I'm jumping the gun on this, so I will revert this!